apps/siyuan/3.1.14/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   siyuan:
-    image: b3log/siyuan:v3.1.14
+    image: b3log/siyuan:v3.1.15
     container_name: ${CONTAINER_NAME}
     restart: unless-stopped
     ports:
```

Comment:
No significant code changes have been identified that require further investigation or improvement. If you need help with specific features of this setup (e.g., installation instructions for additional packages), do not hesitate to ask!